addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -6881,6 +6969,12 @@ registry.ImageTools = ImageHandlerOption.extend({
      */
     async _writeShape(svgText) {
         const img = this._getImg();
+        // Apply the right animation speed if there is an animated shape.
+        const shapeAnimationSpeed = Number(img.dataset.shapeAnimationSpeed) || 0;
+        if (shapeAnimationSpeed) {
+            svgText = this._replaceAnimationDuration(shapeAnimationSpeed, svgText);
+        }
```

Comment:
There's no need since `shapeAnimationSpeed` would be 0 if it's not animated.

Reply:
@bvr-odoo forgot to ping you...

Reply:
> There's no need since `shapeAnimationSpeed` would be 0 if it's not animated.

@SergeBayet You could actually reach this step with 'shapeAnimationSpeed' different from 0 and a not animated shape because the dataset of the image is cleaned only once the shape is selected.